azure-functions-codegen/Cargo.toml

```diff
@@ -15,5 +15,11 @@ proc-macro = true
 azure-functions-shared = { version = "0.3.0", path = "../azure-functions-shared" }
 quote = "0.6.11"
 syn = { version = "0.15.26", features = ["full"] }
-proc-macro2 = { version = "0.4.26", features = ["nightly"] }
+proc-macro2 = { version = "0.4.26" }
 lazy_static = "1.2.0"
+proc-macro-hack = "0.5.4"
```

Comment:
This is a hack by the maintainer of serde so in my mind it's fine to use. It's just a way around the fact that expression based proc-macros are not stable.

Reply:
I'm removing this requirement of expression-based procedural macros when I'm implementing the `cargo func new` command to add an Azure Function to the project.  I'm simplifying the contents of `mod.rs` in the `functions` directory so I can more easily maintain it from a tool.

I'm good with this for now, but hopefully it'll get removed soon.